packages/cerebral-website-tutorial/parts/05_update_state/src/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Controller } from 'cerebral'
+import App from './components/App'
+import { Container } from 'cerebral/react'
+import Devtools from 'cerebral/devtools'
+import { set } from 'cerebral/operators'
```

Comment:
@Guria it's fine it looks like i've figured out some important bits myself
Will create a "windows" branch and a PR for the adjustments i did